functions/controllers/expressExample/index.js

```diff
@@ -25,8 +31,35 @@ app.get("/", async (req, res) => {
   }
 });
 
+app.get("/resend", jwtCheck, async (req, res) => {
+  try {
+    res.json(req.user.sub);
+  } catch (error) {
+    res.status(500).send({ code: 500, message: "Bad/No key" });
+  }
+});
+
 app.post("/resend", jwtCheck, async (req, res) => {
-  res.status(201).send({ message: "private body reached" });
+  // this endpoint takes in a user_id then converts
+  // it into an API call
+  try {
```

Comment:
Prior to sending a request to our API, it's important that we check whether that email we are sending for is currently validated or not. The main reason for this is that even verified users will receive a new verification email. Hypothetically, if someone does get into our API, we want to make sure that they would not be able to spam users who are already verified.